Sources/Paywalls/Components/PaywallTimelineComponent.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallTimelineComponent.swift
+//
+//  Created by mark on 15/1/25.
+
+import Foundation
+
+#if PAYWALL_COMPONENTS
+
+// swiftlint:disable missing_docs nesting
+public extension PaywallComponent {
+
+    struct TimelineComponent: PaywallComponentBase, Equatable, Hashable {
+        let type: ComponentType
+        public let iconAlignment: IconAlignment?
+        public let itemSpacing: CGFloat?
+        public let textSpacing: CGFloat?
+        public let columnGutter: CGFloat?
```

Comment:
Oh interesting... In android I was considering all these mandatory... I guess we could default to 0 in the SDK, but seems like something the backend should always provide? Wdyt?

Reply:
I just made them optional out of abundance of caution 😅 I'd expect the backend to send a default value always yes.

Because of how Decodable works in Swift, I cannot specify a default value, I must define them as optional tho.